docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-sphinx >=4.4
+sphinx ==7.1.2
```

Comment:
to be consistent with rascaline

```suggestion
sphinx < 7.2    # breathe is not compatible with sphinx 7.2 yet
```

Reply:
I tried this first, but it gives me error with furo (it seems to resolve to sphinx 6.2 which also fails)

Reply:
not sure why rascaline is not failing ...

Reply:
okay very nice! lets go for it.